README.md

```diff
@@ -0,0 +1,60 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Review Past Project
+
+> A project to microverse practice Code Review 2.0 training.
+
+## Built With
+
+- HTML
+- CSS
+- VS-Code
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```

Comment:
- [ ] This block of text is not in any way related to the project. The getting started section of your project should contain instructions on how to set up the project locally, including the requirements, prerequisites, and other necessary information.

Reply:
It will be very nice if you add some emojis for your inline comments and reviews, emojis always make your review more human-friendly and make you look like a kind person. :smile:  :hugs: .